### PR TITLE
dumping default consistency description

### PR DESCRIPTION
**Pull request by knull-cn**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

The explanation of `dumping.consistency` is ambiguous, fix it
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@knull-cn Please sign the [Contributor License Agreement](https://cla-assistant.io/pingcap/docs-cn?pullRequest=6668) according to CLAassistant. Thanks. Otherwise, this PR cannot be merged.

**Review by lichunzhu — APPROVED** [review]
LGTM

File: dumpling-overview.md [hunk 1]
```diff
@@ -260,9 +260,9 @@ Dumpling 也可以通过 `-B` 或 `-T` 选项导出特定的数据库/数据表
 
 > **注意：**
 >
-> 在大多数场景下，用户不需要调整 Dumpling 的默认数据一致性选项。
+> 在大多数场景下，用户不需要调整 Dumpling 的默认数据一致性选项(默认值 `auto`)。
```
Comment (TomShawn) [inline dumpling-overview.md:263]:
```suggestion
> 在大多数场景下，用户不需要调整 Dumpling 的默认数据一致性选项（默认值为 `auto`）。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**